x-pack/plugins/observability_onboarding/public/components/app/system_logs/system_integration_banner.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useEffect } from 'react';
+import type { MouseEvent } from 'react';
+import { useKibanaNavigation } from '../../../hooks/use_kibana_navigation';
+import { useSystemIntegrationInstall } from '../../../hooks/use_system_integration_install';
+import { useSystemIntegrationStatus } from '../../../hooks/use_system_integration_status';
+import { PopoverTooltip } from '../../shared/popover_tooltip';
+
+export function SystemIntegrationBanner() {
+  const { navigateToAppUrl } = useKibanaNavigation();
+  const { status: systemIntegrationStatus, data: systemIntegrationData } =
+    useSystemIntegrationStatus();
+  const systemIntegrationInstaller = useSystemIntegrationInstall();
+
+  useEffect(() => {
+    if (systemIntegrationData?.status === 'not_installed') {
+      systemIntegrationInstaller.mutate(systemIntegrationData?.version);
+    }
```

Comment:
I'd prefer if this was the result of a button click instead of happening automatically but I suppose this has already been discussed?

Reply:
Btw. what happens if the install fails? Is this required anywhere in the flow?